HACKING.md

```diff
@@ -1,288 +1,169 @@
-# Goldstone Hacking Guide
+Goldstone Server Hacking Guide
+================
 
-This explains how to install and run Goldstone locally, so you can do code development on it.
 
-The recommended steps are:
+This document explains how to install and run Goldstone Server locally (mostly in docker containers), so you can do code development on the project.  The instructions assume a MacOS X development environment with [homebrew](http://brew.sh/) and [Virtualbox](https://www.virtualbox.org/wiki/Downloads) installed.  The setup process for other environments will be similar, but inevitably be different.
 
-1. Fork this repository on GitHub.
-2. Follow the "Installation" instructions to set up your development environment.
-3. Verify that you can build a working Goldstone application before you make any edits!
-4. Make your improvements in your fork. Remember to include unit tests, and use tox (with the supplied `tox.ini`) to verify that the codebase still passes pep8, pylint, and all unit tests.
-5. Submit a pull request to us. Be sure to include a problem description and an overview of your solution.
+[TOC]
 
-Remember that an instance of Goldstone running locally can be used to monitor a
-local *or remote* OpenStack installation. The server on which Goldstone runs is
-independent from the OpenStack cloud's location.
+Prerequisites
+----------------
 
+Install various prerequisite packages:
 
-## Installation
+    $ brew upgrade
```

Comment:
`$ brew update` ?

From `man brew` : 
update [--rebase]
              Fetch the newest version of Homebrew and all formulae from GitHub using git(1).

upgrade [install-options] [formulae]
              Upgrade outdated, unpinned brews.